ModKit/UI/UI+Toggles.cs

```diff
@@ -82,6 +82,21 @@ public static partial class UI {
             }
             return value;
         }
+
+        public static bool ToggleCallback(
```

Comment:
why not make this an overload of ActionToggle?

Reply:
I considered it, but didn't want to mess with existing methods.